index.html

```diff
@@ -331,7 +331,108 @@ <h3 class="h4">AI tool usage</h3>
       </section>
 
       <!-- FAQS SECTION -->
-      <section id="faqs"></section>
+      <section id="faqs">
+        <h2>
+          <mark class="mark-teal -center">FAQs</mark>
+        </h2>
+        <ul
+          class="faq-list"
+          role="list"
+          aria-roledescription="frequently asked questions"
+        >
+          <li id="faq-1">
+            <details class="faq-details">
```

Comment:
One more thing I just noticed, would it be possible to add some sort of focus style for keyboard users? I'm able to tab through them and open them with a keyboard, but there's currently no visual indication of which question I am on

Reply:
I can't seem to append it to the +/- "button", I think maybe because they're custom markers and are made with pseudo-elements...let me know what you think of this fix.

Reply:
I think as long as the section has an outline then it's fine that the button doesn't also have one

Reply:
Mohammad's on board with this design also, but I have added a bug ticket to revisit the styling or add JS to make the outline only visible to keyboard-navigators with some notes about what I've tried so far that hasn't worked.